pypit/spectrographs/spectro_utils.py

```diff
@@ -0,0 +1,36 @@
+""" Utilities for spectograph codes
+"""
+from __future__ import (print_function, absolute_import, division, unicode_literals)
+import numpy as np
+
+from pypit import msgs
+from pypit.spectrographs import deimos
+from pypit.spectrographs import lris
+from pypit.spectrographs import spectroclass
+
+from pypit import ardebug as debugger
+
+
+def load_spec_class(spectrograph=None, data_file=None):
+    """
+    Instantiate a Spectrograph class based on the given input
+
+    Args:
+        spectrograph: str, optional
+        data_file: str, optional
+          NOT YET IMPLEMENTED
+
+    Returns:
+        spec_class: Spectrograph class (or one of its children)
+
+    """
+    if spectrograph is not None:
```

Comment:
This is great - I advocate that this is the only place in the code now that we should have an `if/elif spectrograph name = blah` command.

Also, we should either make spectrograph a requirement (not an optional argument), or an else statement needs to go with this if statement raising an error that a spectrograph name has not been supplied (or, raise a warning and set `spec_class = spectroclass.Spectrograph()`)

Reply:
again, @jhennawi is going to insist this can be done on a single input file